fahrschein-http-spring/build.gradle

```diff
@@ -14,3 +14,7 @@ dependencies {
 }
 
 publishing.publications.maven.pom.description = 'Fahrschein HTTP Client using Spring Adapter'
+
+tasks.withType(JavaCompile).configureEach {
```

Comment:
the fun fact here: the spring-boot-starter is hard-coded against the apache http client, so this one is actually irrelevant. (and even if not, the spring-boot-starter would be able to use it despite being built with java8)